packages/trader/src/Modules/Contract/Containers/contract-replay.jsx

```diff
@@ -343,15 +345,23 @@ const ReplayChart = observer(({ is_accumulator_contract }) => {
                         is_bottom_widget_visible={isBottomWidgetVisible()}
                     />
                 ))}
-            {!is_beta_chart && is_accumulator_contract && !!markers_array && (
+            {!is_beta_chart && is_accumulator_contract && !!markers_array && !is_reset_contract && (
```

Comment:
@kate-deriv if it's is_accumulator_contract, it cannot be is_reset_contract :)

Reply:
Yep, sorry, missed this, will remove